### PR TITLE
_pins/sobono.json

### LINKED ISSUE #10265: please add me - Richland, WA
Adding Richland, WA to the repo.
Your Latitude: 46.2857
Your Longitude: -119.2845

### PR DESCRIPTION
**Pull request by sobono**
@githubteacher
This closes #10265

### CODE REVIEW

**Comment by sobono** [comment]
Commenting on pull requuest.

File: name & coordinates [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: sobono-RichlandWA
+Latitude: 46.2857
```
Comment (crichID) [inline name & coordinates]: @sobono latitude and longitude must start with lowercase l's instead of captial letters!

**Comment by sobono** [comment]
Re-opened pull request.

File: name & coordinates (hunk 1 above)
Comment (sobono) [reply name & coordinates]: thank you

**Comment by sobono** [comment]
Changed longitude and latitude from upper case to lower case. Changed file name to _pins/sobono.json.

**Comment by crichID** [comment]
@sobono the file name looks like it is still name & coordinates. Can I help you with this change?

**Comment by sobono** [comment]
Sure. Please do. I thought I changed it to this: _pins/sobono.json

**Comment by crichID** [comment]
Done! Let's see what happens with the tests!

**Comment by eafpres** [comment]
This looks OK to me.  What is the issue?

**Pull request merged**